composer.json

```diff
@@ -12,15 +12,25 @@
         "emit"
     ],
     "require-dev": {
-        "phpunit/phpunit": "3.7.*"
+        "phpunit/phpunit": "^5.5",
```

Comment:
Can you change the `phpunit` requirement to `^4.8` ? 
`phpunit` above `5.x` does not work on php5.5, but I want to enable support of this library on 5.5, at least for a while (since there is no reason cutting it yet).